### ISSUE TITLE
Implementation of BPR loss, softplus is different from logsigmoid

### ISSUE THREAD

**Issue by shahjaidev** [gusye1234/LightGCN-PyTorch#30]
As a follow up to the previously asked issue: https://github.com/gusye1234/LightGCN-PyTorch/issues/4

The paper uses logsigmoid in the loss function, but the code uses softplus. Why is this the case? These do not seem to be mathematically equivalent.

https://pytorch.org/docs/stable/generated/torch.nn.Softplus.html

https://pytorch.org/docs/stable/generated/torch.nn.LogSigmoid.html
